include/deal.II/meshworker/assembler.h

```diff
@@ -1023,9 +1023,6 @@ namespace MeshWorker
       const unsigned int level1,
       const unsigned int level2)
     {
-//      AssertDimension(local.n(), dof1.size());
-//      AssertDimension(local.m(), dof2.size());
```

Comment:
This and the other hunk in this file seems useful to retain -- it may in fact be the case that the assertions are valid and should be retained. Can you take this one hunk out of the patch? The rest looks good to remove.